README.md

```diff
@@ -28,17 +28,7 @@ The project runs on Python 3.
 3. Install all the dependencies in `requirements.txt` file:
 `pip install -r requirements.txt`
 
-4. Make sure you exported the following [environment variables](https://github.com/systers/mentorship-backend/wiki/Environment-Variables):
```

Comment:
Can you leave part as it is? so that people at least know where they can find more information about these environment variables. you can just leave the 2 options on step 4.